lib/broccoli/ember-app.js

```diff
@@ -679,23 +674,21 @@ class EmberApp {
     @return {Tree}
   */
   _filterAppTree() {
-    if (this._cachedFilterAppTree) {
-      return this._cachedFilterAppTree;
+    if (!this._cachedFilterAppTree) {
```

Comment:
I'm fine with either, but what I definitely don't like is stuff like `return this.cachedSomething = something;` aka. assignments in return statements. I actually thought I enabled the ESLint rule that prevented stuff like that